src/proposal.ts

```diff
@@ -220,6 +220,47 @@ export type ConsensusParams = {
   };
 };
 
+export type ConsumerParams = {
```

Comment:
why redefining if this struct is exportable from `neutronjs` `src/interchain_security/ccv/v1/shared_consumer.ts`?